test-framework/junit5-mockito/src/main/java/io/quarkus/test/junit/mockito/internal/SingletonToApplicationScopedTestBuildChainCustomizerProducer.java

```diff
@@ -44,12 +47,17 @@ public void accept(BuildChainBuilder buildChainBuilder) {
                     public void execute(BuildContext context) {
                         Set<DotName> mockTypes = new HashSet<>();
                         List<AnnotationInstance> instances = new ArrayList<>();
-                        instances.addAll(testClassesIndex.getAnnotations(INJECT_MOCK));
+                        instances.addAll(testClassesIndex.getAnnotations(DEPRECATED_INJECT_MOCK));
                         instances.addAll(testClassesIndex.getAnnotations(INJECT_SPY));
+                        instances.addAll(testClassesIndex.getAnnotations(MOCKITO_CONFIG));
                         for (AnnotationInstance instance : instances) {
                             if (instance.target().kind() != AnnotationTarget.Kind.FIELD) {
                                 continue;
                             }
+                            if (instance.name().equals(MOCKITO_CONFIG)
```

Comment:
I think there was a typo here maybe this should have been:
```
if (instance.name().equals(MOCKITO_CONFIG) && instance.target().asField().hasAnnotation(DEPRECATED_INJECT_MOCK)) {
   continue;
}
```
As I think that we shouldn't process `@MockConfig` and the deprecated  `@InjectMock` twice as it sets the same configs, but we should process the `@MockConfig` if being used in conjunction with the new `@InjectMock` to tweak the mock configuration.
Since now using `@MockitoConfig` to convert scopes doesn't work =/
Also saw that there is no test for converting scopes, so this it wasn't caught before.
Good thing is that there is still the deprecated `@InjectMock` as we can use it do to scope conversion.